synapse/servers/cortex.py

```diff
@@ -33,27 +33,17 @@ def parse(argv):
 
     s_common.setlogging(logger)
 
-    outp.printf('starting cortex: %s' % (opts.coredir,))
-
-    core = await s_cortex.Cortex.anit(opts.coredir)
-
-    try:
-
-        outp.printf('...cortex API (telepath): %s' % (opts.telepath,))
-        await core.dmon.listen(opts.telepath)
-
-        outp.printf('...cortex API (https): %s' % (opts.port,))
-        await core.addHttpsPort(opts.port)
-
-        if opts.name:
-            outp.printf(f'...cortex additional share name: {opts.name}')
-            core.dmon.share(opts.name, core)
+    core = await s_s_cell.getCell(outp,
```

Comment:
I dont want to funnel cell constructors through a central factory function.  It makes things messy when some cells want custom options.  I dont really see any advantage to having this abstraction in the case for built-in cells :)

Reply:
backed these out